### DISCUSSION TITLE
DetailsList component doesn't keep selected items

### DISCUSSION THREAD

**Discussion by ozan1535** [microsoft/fluentui#26279] [Q&A] [answered]
I want to use DetailsList component and select multiple items. when I click a button on DetailsList header, I want to delete those selected items. I can get selected items, but the problem is when I click header all selected items disappear. You can also see it on the Fluent UI documentation page: 
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/customgroupheaders

Firstly, select some items (or just click `Select group`), then click `Custom header for group 0` . you will see all selected items disappear.

Is there a solution for this problem?

**Comment by spmonahan** [comment, marked as answer]
Hi @ozan1535, try setting `selectionPreservedOnEmptyClick={true}` on `DetailsList`.

**Reply by ozan1535** [reply]
Thank you
